src/snowcli/cli/nativeapp/init.py

```diff
@@ -7,7 +7,6 @@
 from tempfile import TemporaryDirectory
 from click.exceptions import ClickException
 from shutil import move, rmtree
-from git import Repo
```

Comment:
We have similar imports on main branch in `version_processor.py` in native apps code.

I added `print("xxxx")` below these imports in `version_processor.py` and `xxxx` was printed when I executed `snow sql` command, so I think `git` module will be always imported on every command execution.

Reply:
That is ok, but on 1.2.4 not. There should not be native apps :)

Reply:
IMO it's not ok also on `main` because `git` should not be imported if not needed (for example when running `snow sql`).